prow/config.gen.yaml

```diff
@@ -1403,22 +1403,23 @@ presubmits:
           requests:
             cpu: "2"
             memory: 2Gi
-  - name: test-infra_build-containers
+
+  - name: test-infra_build-containers-2.0
     decorate: true
     path_alias: github.com/maistra/test-infra
     skip_report: false
-    run_if_changed: '^docker/maistra-builder_.*\.Dockerfile|^docker/scripts'
+    run_if_changed: '^docker/maistra-builder_2\.0\.Dockerfile|^docker/scripts'
     branches:
       - main
     max_concurrency: 2
     spec:
       containers:
-      - image: "quay.io/maistra-dev/maistra-builder:2.2"
+      - image: "quay.io/maistra-dev/maistra-builder:2.0"
```

Comment:
do we need to adjust this? I think the test-infra jobs should always run using the latest image

Reply:
Either way should work, I think. The benefit of doing this way is that it's easier for maintenance. Having the numbers aligned and also not having to bump the version when the latest version bumps... Not a strong preference though.